### DISCUSSION TITLE
Chat failed to load. Please ensure that the GitHub Copilot Chat extension is up to date.

### DISCUSSION THREAD

**Discussion by RezaShamsipour** [community/community#123526] [Copilot Conversations]
### Select Topic Area

Bug

### Body

My Copilot Version : 1.190.0
Copilot Chat : 0.15.1

i get error after auto updataed to latest version.

**Comment by slamdunkyc** [comment]
Got same issue here, works fine after install the prevoius version `1.188.0`

**Reply by RezaShamsipour** [reply]
Only Change update for Copilot ? also dont work....
